README.md

```diff
@@ -11,3 +11,7 @@ A repository of deep learning examples.
 ## Examples
 
 - [YoloV5](yolov5/)
+        - `cd yolov5`
+        - `git clone git@github.com:ultralytics/yolov5.git`
```

Comment:
clone from https url as that's what most people are used to

Reply:
I don't think this is really necessary